### PR TITLE
Lighting

### PR DESCRIPTION
**Pull request by timoffex**
Added lighting.

### CODE REVIEW

File: WallsAndHoles/meshview.h.orig [hunk 1]
```diff
@@ -0,0 +1,94 @@
+#ifndef MESHVIEW_H
```
Comment (Bdtrotte) [inline WallsAndHoles/meshview.h.orig]: Stray file? Git rid of it!

File: WallsAndHoles/mainwindow.h [hunk 2]
```diff
@@ -34,6 +34,9 @@ private slots:
 
     void on_actionimport_triggered();
 
+protected:
+    MeshViewContainer *mMeshViewContainer;
```
Comment (Bdtrotte) [inline WallsAndHoles/mainwindow.h]: Why is this here? Main window should remain empty. (MeshViewContainer is added as a dock widget by Editor)

File: WallsAndHoles/meshviewcontainer.cpp [hunk 3]
```diff
@@ -20,7 +20,7 @@ MeshViewContainer::MeshViewContainer(QWidget *parent) :
     mMeshView = findChild<MeshView *>();
     //mMeshView->setScene(scene);
 
-    mMeshView->load(":/assets/cube.obj");
+    mMeshView->load(":/assets/teapot.obj");
```
Comment (Bdtrotte) [inline WallsAndHoles/meshviewcontainer.cpp:23]: Still a bad cube

File: WallsAndHoles/meshviewcontainer.cpp.orig [hunk 4]
```diff
@@ -0,0 +1,53 @@
+#include "meshviewcontainer.h"
```
Comment (Bdtrotte) [inline WallsAndHoles/meshviewcontainer.cpp.orig]: Another stray file?

**Review by Bdtrotte — CHANGES_REQUESTED** [review]
Some stray files and possibly lines. Good stuff otherwise!

File: WallsAndHoles/meshviewcontainer.cpp (hunk 3 above)
Comment (timoffex) [reply WallsAndHoles/meshviewcontainer.cpp:23]: We're not making Minecraft
